tests/test_agents/test_apy_estimation_abci.py

```diff
@@ -54,19 +54,14 @@ class BaseTestABCIAPYEstimationSkillNormalExecution(BaseTestEnd2EndExecution):
     agent_package = "valory/apy_estimation_chained:0.1.0"
     skill_package = "valory/apy_estimation_chained_abci:0.1.0"
     happy_path = HAPPY_PATH
-    ROUND_TIMEOUT_SECONDS = 120
-    wait_to_finish = 240
+    ROUND_TIMEOUT_SECONDS = 480
+    wait_to_finish = 480
```

Comment:
Increased round timeout to try to resolve APY 4 agents' e2e flakiness caused by the optimization not finishing.